### ISSUE TITLE
Cant get Users.

### ISSUE THREAD

**Issue by dimandus** [Discord-Net-Labs/Discord.Net-Labs#264]
GetUserAsync returns null
SocketGuild.DownloadUsersAsync infinite awaiting
Maybe some another problems.

I switched to Labs from regular Discord.Net for the functionality of the buttons, but some old code stopped working.
The bot version on Discord.Net is working successfully.

Example of my code
`SocketGuild guild = ((SocketGuildChannel)this.source.Channel).Guild;
await guild.DownloadUsersAsync();`

**Comment by quinchs** [comment]
You need to specify gateway intents, you can do this by adding them to your client config. You will also need to enable the guild member intent on the developer portal for your bot

**Comment by dimandus** [comment]
> You need to specify gateway intents, you can do this by adding them to your client config. You will also need to enable the guild member intent on the developer portal for your bot

its all enabled.
![image](https://user-images.githubusercontent.com/15235058/139296553-ddcce6f7-9e92-4b59-8243-9f6e9cc0e079.png)

And again, the Discord.Net version of the bot works without problems.
Perhaps Discord.Net.Labs needs additional configuration?
I just changed the framework via NuGet and some code stopped working.

UPD:
I added `GatewayIntents = GatewayIntents.All` to DiscordSocketConfig and all works.
Don't know why Discord.Net bot works fine without it at the same time now.
But Labs maybe need this.
